homeassistant/components/aurora/coordinator.py

```diff
@@ -38,8 +38,8 @@ def __init__(self, hass: HomeAssistant) -> None:
         )
 
         self.api = AuroraForecast(async_get_clientsession(hass))
-        self.latitude = int(self.config_entry.data[CONF_LATITUDE])
-        self.longitude = int(self.config_entry.data[CONF_LONGITUDE])
+        self.latitude = round(self.config_entry.data[CONF_LATITUDE])
+        self.longitude = round(self.config_entry.data[CONF_LONGITUDE])
```

Comment:
Doesn't rounding also make it an integer? I thought the gist of the issue was that aurora would not be precise enough because it removed everything fractional

Reply:
Hi,
The resolution of the data that is fetched from "https://services.swpc.noaa.gov/json/ovation_aurora_latest.json" is only per integer degree of latitude and longitude. The problem as I understood it is that when giving e.g. a value say 17.9 you would read the data for "17" instead of "18" which is more appropriate.
/Cheers Johan

Reply:
Okay I think that it's worth an explanation in the issue so the rest knows that it shouldn't be a float

Reply:
Ok, I will add a comment with an explanation in the issue thread.